app-backend/db/src/test/scala/com/azavea/rf/database/CategoryToTemplateDaoSpec.scala

```diff
@@ -11,5 +11,5 @@ import doobie.scalatest.imports._
 import org.scalatest._
 
 class CategoryToTemplateDaoSpec extends FunSuite with Matchers with IOChecker with DBTestConfig {
-  test("selection types") { check(CategoryToTemplateDao.selectF.query[CategoryToTemplate]) }
+  // test("selection types") { check(CategoryToTemplateDao.selectF.query[CategoryToTemplate]) }
```

Comment:
You can run this simple type check as `CategoryToTemplateDao.query.list.transact(xa).unsafeRunSync`, just outside the `check(...)` call.